doc/ch-grow_desc.rst

```diff
@@ -402,7 +414,7 @@ Download the Debian Buster image and place it in the storage directory::
 
   $ ch-grow pull debian:buster
   pulling image:   debian:buster
-
+  fat manifest: downloading
```

Comment:
This should IMO say which architecture was selected.

Reply:
Before we select the architecture we need to fat manifest. Here we are telling the user it is being downloaded. We could say something like "image platform index: downloading" or something of the like to differentiate it from the term 'manifest'. But fat manifest is technically correct.